docs/add_service_monitoring.md

```diff
@@ -0,0 +1,53 @@
+# Enable monitoring for your service
+
+We use [ODH](https://opendatahub.io/) Prometheus as part of our monitoring stack to scrape metrics.
+
+To enable our Prometheus to monitor your services, you need to add the appropriate RBAC and service monitors.
+
+You can choose to have one servicemonitor per service, or group all your services into one service monitors. The steps are similar in both cases.
+
+### Pre-requisite:
+
+You will need pre-requisite tools to follow along with this doc, please do one of the following:
+
+- Install our [toolbox](https://github.com/operate-first/toolbox) to have the developer setup ready automatically for you.
+- Install the tools manually. You'll need [kustomize](https://kustomize.io/), [sops](https://github.com/mozilla/sops) and [ksops](https://github.com/viaduct-ai/kustomize-sops).
+- Ensure that each namespace the services belong to are listed [here](https://github.com/operate-first/apps/tree/master/cluster-scope/base/namespaces). If not then please file an issue [here](https://github.com/operate-first/support/issues/new?assignees=&labels=onboarding&template=onboarding_to_cluster.md&title=).
+
+Please fork/clone the [operate-first/apps](https://github.com/operate-first/apps) repository. **During this whole setup, we'll be working within this repository.**
+
+### Steps:
+1. Add the following lines to each namespace's `kustomization.yaml` to enable rbac for each namespace:
+```
+components:
+  - ../../../components/project-admin-rolebindings/operate-first
```

Comment:
Is the operate-first really needed in here? I think the `monitoring-rbac` should be enough

Reply:
good call, fixed